src/content/getting-started/create-workload-cluster/index.md

```diff
@@ -66,6 +76,8 @@ kubectl gs template cluster \
   > cluster.yaml
 ```
 
+For backward compatibility, vintage cluster templating does not require the `--name` parameter. If no name is specified, a random one like `zekfaewnxh` will be generated. The `--name` parameter becomes required with CAPI clusters, but you can also use `--generate-name` if a random name is really desired.
```

Comment:
Why we introduce `--generate-name` if the default behaviour is that? do we plan to require `name`?

Reply:
`--name` is now required for CAPI (see https://github.com/giantswarm/roadmap/issues/2983). For vintage, we remain compatible with old behavior i.e. don't require any of the two parameters and default to random names.